src/content/snapshotOptions/delay.mdx

```diff
@@ -15,58 +15,64 @@ Components sometimes trigger custom interactions on render. For example, JavaScr
 
 ## Customizing snapshot delays
 
-Chromatic's maximum wait time before capturing a snapshot is 15 seconds, providing a balance for your tests to load resources and be ready for snapshotting. If you need to customize the wait time for Chromatic to capture a snapshot, add the `delay` configuration option to your tests. For example:
+Chromatic has a multistage timeout for capturing a snapshot: 15 seconds to render a story and an additional 15 seconds to execute interaction tests, if present. This timing provides a balance for your tests to load resources and be ready for snapshotting. If you need to customize the wait time for Chromatic to capture a snapshot, add the `delay` configuration option to your tests. For example:
```

Comment:
Just a thought and feel free to disregard, but I was wondering now that the multistage timeout is in effect, we use some of the wording here and apply it to the snapshot's page, specifically in https://www.chromatic.com/docs/snapshots/#delays

Reply:
Good question, let me follow up with capture and I'll update that in a future PR.